conf/setup.sql

```diff
@@ -1,5 +1,5 @@
 # Create the DB
-CREATE DATABASE css;
+CREATE DATABASE css character set UTF8mb4 collate utf8mb4_bin;
```

Comment:
I don't believe I had to change anything in the mysql conf but I can do more research and see if using the utf8 will have any other issues.